TESTS/mbed_hal/watchdog_reset/main.cpp

```diff
@@ -42,6 +42,8 @@
  */
 #define KICK_ADVANCE_MS 35UL
 
+#define TIMEOUT_LOWER_LIMIT 1000UL
```

Comment:
Please keep the convention of adding a unit suffix.
```suggestion
#define TIMEOUT_LOWER_LIMIT_MS 1000UL
```

Reply:
Done